joomla/joomla-cms
# [4.0] Add missing language strings

### Summary of Changes
Add missing language strings


### Testing Instructions
Create a custom media field.
Edit an article.
Click Fields tab.
See labels not translated.
Repeat on the frontend.
